src/pages/de/guides/markdown-content.mdx

```diff
@@ -4,6 +4,8 @@ title: Markdown
 description: Verwendung von Markdown mit Astro
 ---
 
+import Since from '~/components/Since.astro'
```

Comment:
Wrong language, this is the German file 😅

Reply:
Verdammt